src/orion/core/cli/hunt.py

```diff
@@ -50,5 +59,6 @@ def main(args):
     """Build experiment and execute hunt command"""
     args['root'] = None
     args['leafs'] = []
+    worker_trials = args.pop('worker_trials')
```

Comment:
Current [build](https://travis-ci.org/mila-udem/orion/builds/411819932) fails because of this line. The default value of ```worker_trials``` that is set within ```resolve_config``` actually takes place inside ```EVCBuilder().build_from(args)```. I know this is confusing, I'm currently preparing a PR for refactoring the configuration process. I'll adapt it when I make this PR.

For now you can fix with
```python
from orion.core.io.experiment_builder import ExperimentBuilder

# ... 

worker_trials = ExperimentBuilder().fetch_full_config(args)['worker_trials']
```